src/transformState.js

```diff
@@ -49,7 +49,29 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const prop of transforms) {
+    const { operation, properties } = prop;
```

Comment:
в деяких випадках в тебе не буде properties, в таких випадках добре задавати default значення.
Це робиться для того щоб, якщо нічого немає в нас не поламалась програма.
```suggestion
    const { operation = "", properties = {} } = prop;
```